wbooks/App.js

```diff
@@ -20,14 +20,13 @@ import {
   Colors,
 } from 'react-native/Libraries/NewAppScreen';
 import BookList from './src/app/screens/booklist';
-import styles from './styles';
 
 const App: () => React$Node = () => {
   return (
     <>
-      <StatusBar barStyle="dark-content" />
+      <StatusBar  />
       <SafeAreaView>
-        <View style={styles.container}>
+        <View style={{ height:'100%' }}>
```

Comment:
The styles like `{height: '100%'}` should be in a `styles` file.

Reply:
done